internal/services/conditionalaccess/migrations/conditional_access_policy_resource.go

```diff
@@ -330,6 +330,11 @@ func ResourceConditionalAccessPolicyInstanceResourceV0() *pluginsdk.Resource {
 								Type: pluginsdk.TypeString,
 							},
 						},
+
+						"insider_risk_levels": {
+							Type:     pluginsdk.TypeString,
+							Optional: true,
+						},
```

Comment:
State migrations reference the schema at a specific point in time, so we don't need to add this here
```suggestion
```